.github/workflows/continuous-integration-linux.yml

```diff
@@ -56,7 +56,28 @@ jobs:
             cmake_build_type: 'RelWithDebInfo'
             backend: 'THREADS'
             stdcxx: '23'
-    runs-on: ubuntu-latest
+          - distro: 'ubuntu:latest'
+            cxx: 'g++'
+            cmake_build_type: 'RelWithDebInfo'
+            backend: 'OPENMP'
+            stdcxx: '17'
+            arch: 'arm64'
+            cmake_extra_opts: '-DKokkos_ARCH_NATIVE=ON'
```

Comment:
I am also a bit confused why this is not on by default for all the CI builds.

Reply:
I think it was mostly a question of testing configurations with and without `ARCH_NATIVE` and it didn't seem very relevant for the linux builds that we tested before.

Reply:
Ok, makes sense!